Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base.yml

```diff
@@ -4,7 +4,7 @@
   abstract: true
   components:
   - type: AccessReader
-  - type: SecureEntityStorage
+  - type: Lock
```

Comment:
you  replaced all of these with lock, but didn't add in a normal EntityStorage component as well so I think this breaks lockers?

Reply:
I'll take a look at lockers